csrc/executor_kernel_arg.h

```diff
@@ -263,15 +261,38 @@ template <typename TENSOR_TYPE>
 struct TensorArg : public TensorArgAbstract {
   TENSOR_TYPE instance_;
 
-  TensorArg(const at::Tensor& tensor) : TensorArgAbstract(tensor) {
+  TensorArg(const at::Tensor& tensor, TensorView* tv)
```

Comment:
Is `tv` used just for error checking at line 279? When can it be just `nullptr`?

Reply:
For now, yes, it is just for error checking. And in the next PR, it will be used to infer stride of rFactor domain from allocation domain. It can be nullptr in `FusionExecutor::runRtc` where we don't have a fusion to compile or run.

Reply:
Added